### DISCUSSION TITLE
I'm unable to deploy Airbyte on Local windows system

### DISCUSSION THREAD

**Discussion by kavyapothula14** [airbytehq/airbyte#34469] [General]
I have attached the logs for temporal and server[airbyte-server.txt](https://github.com/airbytehq/airbyte/files/14032201/airbyte-server.txt)
[airbyte-temporal.txt](https://github.com/airbytehq/airbyte/files/14032202/airbyte-temporal.txt)

**Comment by sajarin** [comment]
Hey @kavyapothula14 can you describe your setup on Windows in more detail? Are you using WSL2?

[a comment by an automated account is not shown]
